docs/usage/security-guide.md

```diff
@@ -0,0 +1,60 @@
+# Security guide
+
+Rahti applications are exposed to the Internet and
+their security should be treated with an appropriate care.
+The user on whose account a service is running in Rahti, is
+responsible of its security. See [Terms of Service]() for details about
```

Comment:
I'll create an issue so that we remember to update the link once the relevant docs have been merged to master.